Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/WMFTitleOverlayModel.h

```diff
@@ -0,0 +1,17 @@
+//  Created by Monte Hurd on 7/30/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <Foundation/Foundation.h>
+
+@class WMFTitleOverlayLabel;
+@interface WMFTitleOverlayModel : NSObject
+
+@property (nonatomic, strong) NSLayoutConstraint* topConstraint;
+@property (nonatomic, strong) NSString* anchor;
+@property (nonatomic, strong) NSString* title;
+@property (nonatomic) CGFloat yOffset;
+@property (nonatomic) NSNumber* sectionId;
+
+@property (nonatomic, strong) WMFTitleOverlayLabel* label;
```

Comment:
why does a model have a reference to a label? If we're going to break MVC, there should at least be clear documentation for the rationale.